modules/installation-and-upgrade/pages/proxy-container-k3s-installation.adoc

```diff
@@ -0,0 +1,129 @@
+
+----
+export KUBECONFIG=/path/to/kubeconfig-k3s.yaml
+----
+
+[[installation-proxy-containers-k3s-helm]]
+== Installing helm
+
+[NOTE]
+====
+The Containers Module is required to install `helm`.
+====
+
+Simply run:
+
+    zypper in helm
+
+[[installation-proxy-containers-k3s-metallb]]
+== Installing [literal]``metalLB``
+
+[literal]``MetalLB`` is the load balancer that will expose the {productname} proxy pod services to the outside world.
+To install it, run:
+
+----
+helm repo add metallb https://metallb.github.io/metallb
+helm install --create-namespace -n metallb metallb metallb/metallb 
+----
+
+[literal]``MetalLB`` still requires a configuration to know the virtual IP address range to be used.
+In this example, the virtual IP addresses will be from [literal]``192.168.122.240`` to [literal]``192.168.122.250``, but that range could be lowered to a single address if the host only exposes the {productname} proxy.
+These addresses need to be a subset of the server network.
+
+Create a [path]``metallb-config.yaml`` with content like the following with an IP address range that aligns with the deployed network:
+
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: l2-pool
+  namespace: metallb
+spec:
+  addresses:
+  - 192.168.122.240-192.168.122.250
+---
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: l2 
+  namespace: metallb
+spec:
+  ipAddressPools:
+  - l2-pool
+----
+
+Apply this configuration by running:
+
+----
+kubectl apply -f metallb-config.yaml
+----
+
+
+[[installation-proxy-containers-k3s-deploy]]
+== Deploying the {productname} proxy helm chart
+
+Create a configuration file forcing the IP address that [literal]``MetalLB`` will use for the {productname} Proxy services.
+This IP address needs to be the one to which the proxy FQDN entered when creating the proxy configuration.
+It also needs to be resolvable from both the {productname} server and the systems to connect to the proxy.
+
+This example will use [literal]``192.168.122.241``.
+
+Create a [path]``custom-values.yaml`` file with the following content.
+If the [literal]``MetalLB`` IP address range only contains a single address, the last line can be removed.
+
+----
+services:
+  annotations:
+    metallb.universe.tf/allow-shared-ip: key-to-share-ip
+    metallb.universe.tf/loadBalancerIPs: 192.168.122.241
+----
+
+To configure the storage of the volumes to be used by the {productname} Proxy pod, define persistent volumes for the following claims.
+Please refer to the https://kubernetes.io/docs/concepts/storage/persistent-volumes/[[literal]``kubernetes``] or https://rancher.com/docs/k3s/latest/en/storage/[k3s] documentation for more details.
+The persistent volume claims are named:
+
+* [literal]``squid-cache-pv-claim``
+* [literal]``/package-cache-pv-claim``
+* [literal]``/tftp-boot-pv-claim``
+
+Create the configuration for the {productname} Proxy as documented in xref:proxy-container-setup.adoc[].
```

Comment:
I rather vote to repeat (copy and paste) the configuration here again.  Or we could ask Ornela to make this a snippet and reuse it two times.

Reply:
I would rather have a snippet than duplicating the data as it makes it easier to maintain in the long run.

Reply:
ok, for the moment we can keep it as-is (imo).